pages/docs/couple-your-code/couple-your-code-waveform.md

```diff
@@ -35,27 +35,26 @@ Linear interpolation between coupling boundary conditions of the previous and th
 
 If the Dirichlet participant $$\mathcal{D}$$ calls `readData`, it samples the data from a time-dependent function $$c_\mathcal{D}(t)$$. This function is created from linear interpolation of the first and the last sample $$c_{\mathcal{D}0}$$ and $$c_{\mathcal{D}5}$$ created by the Neumann participant $$\mathcal{N}$$ in the current time window. This allows $$\mathcal{D}$$ to sample the coupling condition at arbitrary times $$t$$ inside the current time window.
 
-{% note %}
-As soon as time interpolation is used to obtain data for a `relativeReadTime` that does not refer to the end of the current window, the data from the end of the previous window might be used to compute the interpolated value. For the first time window, this means one should provide appropriate initial data, since otherwise preCICE will use zero-valued initial data. There are, however, situations where initial data is not necessarily needed. For example, if one uses zeroth order time interpolation without accessing the data value at the beginning of the time window. It is still recommended to always provide initial data, if available. See ["Step 7 - Data initialization"](couple-your-code-initializing-coupling-data.md) for details on data initialization.
-{% endnote %}
+## API for waveform iteration
```

Comment:
This sounds like there is a dedicated API for the waveform iteration, which isn't the case.

Maybe `Using waveform iteration` or `In the adapter` or `In practise` would be easier to understand.

Reply:
`Time interpolation in practise` :wink: